spec/helpers.rb

```diff
@@ -0,0 +1,5 @@
+module Helpers
+  def card_new(orig, transl) # new card without .save
+    Card.new(original_text: orig, translated_text: transl, review_date: DateTime.now)
```

Comment:
Line is too long. [85/80]<br>Do not use `DateTime.now` without zone. Use one of `Time.zone.now`, `DateTime.now.current`, `DateTime.now.in_time_zone`, `DateTime.now.utc`, `DateTime.now.getlocal`, `DateTime.now.iso8601`, `DateTime.now.jisx0301`, `DateTime.now.rfc3339`, `DateTime.now.to_i`, `DateTime.now.to_f` instead.